### PR TITLE
BDF query -- POC for tracking page views within DaaS

### PR DESCRIPTION
**Pull request by langswei**
Please ensure your pull request adheres to the following guidelines:
- [ ] make sure to link the related issues in this description
- [ ] when merging / squashing, make sure the fixed issue references are visible in the commits, for easy compilation of release notes

## Related Issues

Thanks for contributing!

### CODE REVIEW

File: src/queries/rum-pageviews-bdf.sql [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+--- startdate: 2023-05-01
+--- enddate: 2023-06-01
+--- timezone: UTC
+--- url: -
+--- device: all
+--- domainkey: secret
+
+WITH rum AS (
+  SELECT
+    REGEXP_REPLACE(hostname, 'www.', '') AS hostname,
```
Comment (trieloff) [inline src/queries/rum-pageviews-bdf.sql:14]:
```suggestion
    REGEXP_REPLACE(hostname, r'$www.', '') AS hostname,
```

File: src/queries/rum-pageviews-bdf.sql [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    COUNT(DISTINCT id) AS rum_count,
```
Comment (trieloff) [inline src/queries/rum-pageviews-bdf.sql:15]:
```suggestion
    COUNT(DISTINCT id) AS ids,
```

just for consistency with the other queries.

File: src/queries/rum-pageviews-bdf.sql [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    weight,
+    FORMAT_DATE('%Y-%m-%d', time) AS date
```
Comment (trieloff) [inline src/queries/rum-pageviews-bdf.sql:17]:
```suggestion
    FORMAT_DATE('%F', time) AS date
```


%F | DATEDATETIMETIMESTAMP | The date in the format %Y-%m-%d.
-- | -- | --





%F | DATEDATETIMETIMESTAMP | The date in the format %Y-%m-%d.
-- | -- | --

File: src/queries/rum-pageviews-bdf.sql [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    @domainkey
+  )
+  GROUP BY date, weight, hostname
+)
+SELECT
+  a.hostname,
+  SUM(a.rum_count * a.weight) AS estimated_pv,
+  a.date,
+  b.ims_org_id
+FROM rum a
```
Comment (trieloff) [inline src/queries/rum-pageviews-bdf.sql:35]:
```suggestion
FROM rum AS rum_data
```

This makes it a bit easier to understand what's going on then `a` and `b`

File: src/queries/rum-pageviews-bdf.sql [hunk 5]
```diff
@@ -0,0 +1,46 @@
+--- description: Show page views for specified domains for specified dates based on extrapolation from RUM data
```
Comment (trieloff) [inline src/queries/rum-pageviews-bdf.sql:1]: I'm not sure what BDF means and this would be a good place to include it.

**Comment by lydiapuric** [comment]
This POC is outdated and this PR can be closed without merge.

**Pull request closed without merging**